packages/opentelemetry-semantic-conventions-ai/opentelemetry/semconv/ai/__init__.py

```diff
@@ -46,6 +46,18 @@ class SpanAttributes:
 
 class EventName(Enum):
     VECTOR_DB_QUERY_EMBEDDINGS = "vector_db.query.embeddings"
+    VECTOR_DB_QUERY_RESULT = "vector_db.query.result"
+
+
+class EmbeddingsEventAttributes(Enum):
+    VECTOR = "vector_db.query.embeddings.vector"
```

Comment:
Let's use the same notation we had for SpanAttributes - all are in the same enum